tests/filecheck/mlir-conversion/with-mlir/dialects/emitc/emitc_types.mlir

```diff
@@ -2,11 +2,13 @@
 
 // CHECK: f32_2D = !emitc.array<4x2xf32>
 // CHECK-SAME: f64_1D = !emitc.array<5xf64>
+// CHECK-SAME: i1_0D = !emitc.array<0xi1>
 // CHECK-SAME: i1_3D = !emitc.array<3x4x5xi1>
 // CHECK-SAME: i32_1D = !emitc.array<10xi32>
```

Comment:
Can you please order these so they're in the same order as the input? Not a big deal but threw me off slightly

Reply:
yeah mlir-opt orders them, I left the original ordering in the test to bring this up